tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -277,8 +288,43 @@ <h3>No scalar data was found.</h3>
         return _smoothingWeight > 0;
       },
 
-      _shouldOpen(index) {
-        return index <= 2;
+      _getCategoryKey(category) {
+        return category.metadata.type ==
+            tf_categorization_utils.CategoryType.SEARCH_RESULTS ?
+                'search' : category.name;
+      },
+
+      _updatePaneOpened() {
+        const paneOpen = this._paneOpened;
+        const nextPaneOpen = {};
+
+        // Copy over the relevant values from the old pane state.
+        this._categories
+            .map(this._getCategoryKey)
+            .filter(name => paneOpen[name] != null)
+            .forEach(name => nextPaneOpen[name] = paneOpen[name]);
+
+        if (Object.keys(nextPaneOpen).length == 0) {
+          this._categories.slice(0, 2)
+              .map(this._getCategoryKey)
+              .forEach(name => {
+                nextPaneOpen[name] = nextPaneOpen[name] != null ?
```

Comment:
Can't this just be `nextPaneOpen[name] = true`?  This is all guarded by `Object.keys(nextPaneOpen).length == 0` so I would take that to mean that `nextPaneOpen[name] == null` is always true, making the ternary redundant.